backend/IDE.Common/ModelsDTO/Enums/ProjectType.cs

```diff
@@ -4,6 +4,8 @@ public enum ProjectType : int
     {
         Console = 0,
         Web = 1,
-        Library = 2
+        Library = 2,
+        JsTsApp = 3,
```

Comment:
I don't think this is the right way to set ProjectType as a JS/TS or GoApp type. The ProjectType entity should simply describe the type of project (i.e. mobile/web/library/console)

Reply:
Ok, Can you explain what project type should be for Js/Ts and Go. Now I have:
C# Console, Web, Library;
For Js/Ts I just left Js/Ts App;
For Go - GoApp;